client/src/components/Toast.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+toast.configure();
+
+function Toast() {
```

Comment:
vllt noch umbennen in FehlerErstelltToast oder so, der scheint ja nur dafür zu sein.

Reply:
man könnte es auch ein "SuccessToast" und die Nachricht "Fehler erstellt." als Parameter der funktion übergeben